amr-wind/equation_systems/DiffusionOps.H

```diff
@@ -43,6 +43,8 @@ public:
     virtual void linsys_solve(const amrex::Real dt);
 
 protected:
+    virtual void linsys_solve_impl();
```

Comment:
So we need this to be a public method for CUDA.

Reply:
Done in https://github.com/Exawind/amr-wind/pull/322/commits/03d22f369176cc4c56cae88622ea4da2dbb4d446